ini/stm32f4.ini

```diff
@@ -838,3 +838,22 @@ build_flags                 = ${stm32_variant.build_flags}
                               -DUSBD_IRQ_PRIO=5 -DUSBD_IRQ_SUBPRIO=6
                               -DHSE_VALUE=8000000U
 upload_protocol             = stlink
+
+#
+# Mellow Fly E3 Pro V3 (STM32F407VGT6 ARM Cortex-M4)
+#
+[env:FLY_E3_PRO_V3]
+extends                     = stm32_variant
+board                       = marlin_STM32F407VGT6_CCM
+board_build.variant         = MARLIN_F4x7Vx
+board_build.offset          = 0x8000
+board_upload.offset_address = 0x08008000
+build_flags                 = ${stm32_variant.build_flags}
+                              -DHAVE_HWSERIAL1 -DHAVE_HWSERIAL3
+                              -DPIN_SERIAL1_RX=PA_10 -DPIN_SERIAL1_TX=PA_9
+                              -DPIN_SERIAL3_RX=PD_9 -DPIN_SERIAL3_TX=PD_8
+                              -DHAL_SD_MODULE_ENABLED
+                              -DUSE_USBHOST_HS -DUSE_USB_HS_IN_FS
+                              -DUSBD_IRQ_PRIO=5 -DUSBD_IRQ_SUBPRIO=6
+                              -DHSE_VALUE=8000000U
+upload_protocol             = stlink
```

Comment:
@jaysuk, this appears to be an exact duplicate of the FLY_E3_V2 environment. Was there a reason to make a new one rather than just re-using the old? Are you expecting additional changes that would cause them to diverge?

Reply:
didn't know that was a think I could do as every board seems to do things in a different way

Reply:
And the temp sensors come installed to the board if you purchase that option

Reply:
I'd recommend deleting the duped environment & rename `FLY_E3_V2` to something like `STM32F407VG_mellow_fly` (and update the related `env:`s in `pins.h` as well as `renamed.ini`).